atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -313,7 +313,8 @@ def _get_site_mapping(self, registry):
         elif mapping_file in self.file_package_mappings:
             return self.file_package_mappings[mapping_file]
 
-        mapping = read_yaml_from_file_path(mapping_file, "schemas/package_mapping.json")
+        self.log.debug("Downloading mapping file for %s", registry)  # TODO: safe to log URL?
```

Comment:
Not sure about this, WDYT?

Reply:
It should be fine, we have nothing to hide :D  (unless URL contains password, but we are good engineers and we don't do that)